2015/music-and-measure-theory/german/sentence_translations.json

```diff
@@ -20,34 +20,34 @@
  {
   "input": "The second challenge, which I'll get to about halfway through the video, has to do with covering numbers with open sets, and is very counterintuitive.",
   "model": "nmt",
-  "translatedText": "Die zweite Herausforderung, auf die ich etwa in der Mitte des Videos eingehen werde, besteht darin, Zahlen mit offenen Mengen abzudecken, und ist sehr kontraintuitiv.",
+  "translatedText": "Die zweite Aufgabe, auf die ich etwa in der Mitte des Videos eingehen werde, besteht darin, Zahlen mit offenen Mengen abzudecken, und ist sehr kontraintuitiv.",
   "time_range": [
    18.06,
    25.24
   ]
  },
- {
-  "input": "Or at least, when I first saw it, I was confused for a while.",
-  "model": "nmt",
-  "translatedText": "Zumindest war ich eine Weile verwirrt, als ich es zum ersten Mal sah.",
-  "time_range": [
-   25.86,
-   28.54
-  ]
- },
- {
-  "input": "Foremost, I'd like to explain what's going on, but I also plan to share a surprising connection that it has with music.",
-  "model": "nmt",
-  "translatedText": "Zunächst möchte ich erklären, worum es geht, aber ich möchte auch auf eine überraschende Verbindung hinweisen, die es mit der Musik hat.",
-  "time_range": [
-   29.28,
-   35.16
-  ]
- },
+  {
+    "input": "Or at least, when I first saw it, I was confused for a while.",
+    "model": "nmt",
+    "translatedText": "Zumindest, als ich es zum ersten Mal sah, war ich eine Weile verwirrt.",
```

Comment:
Uncommon to say this in German like that. Suggestion:
`war ich etwas verwirrt.`

Reply:
fixed in latest commit